### PR TITLE
YALB-474: Controls refactor

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
### Description of work
- Refactors all previous uses of the cta component that should not have had all of the cta markup

### Testing Link(s)
- [ ] Navigate to the [Component Story](https://deploy-preview-112--dev-component-library-twig.netlify.app/?path=/story/tokens-breakpoints--breakpoints)

### Functional Review Steps
- [ ] Verify everything still looks and functions as expected.

### CODE REVIEW

File: components/01-atoms/controls/base/control.twig [hunk 1]
```diff
@@ -8,6 +12,15 @@
     href: control__url,
   }) %}
   {% set control__element = 'a' %}
+{% else %}
+  {% set control__base_class = control__base_class|default('button') %}
+  {% set control__element = control__element|default('button') %}
+{% endif %}
+
+{% if aria_expanded %}
```
Comment (callinmullaney) [inline components/01-atoms/controls/base/control.twig:20]: `aria_expanded` should only ever apply to a `<button>` correct? I could see this being moved up inside the else but I guess it doesn't need to be.

**Review by callinmullaney — APPROVED** [review]
@ModulesUnraveled Left a small code comment but in general this PR looks really good and simplifies the implementation of cta/control quite a bit. Nice work :+1:

**Review by miketullo95 — APPROVED** [review]
👍  looks great!

File: components/01-atoms/controls/base/control.twig (hunk 1 above)
Comment (ModulesUnraveled) [reply components/01-atoms/controls/base/control.twig]: Makes sense. I moved it in and made sure everything still worked as expected. Thanks!

**Pull request merged**